wrappingGsdk/README.md

```diff
@@ -28,7 +28,7 @@ dotnet publish --self-contained -r win-x64 /p:PublishSingleFile=true /p:PublishT
 
 The `wrapper` executable (as well as its debugging symbols) will be published into `wrapper\bin\Release\net8.0\win-x64\publish` directory. Next step would be to build your game server executable and package it along with the `wrapper` output.
 
-As mentioned, if you're just evaluating the platform or don't have a game server of your own, you can use `fakegame` sample. In this case, we have provided a convenient script [build.ps1](./build.ps1) that will build and package both projects (`wrapper` and `fakegame`). Script will create a `drop` folder with a .zip file containing the required files. 
+As mentioned, if you're just evaluating the platform or don't have a game server of your own, you can use `fakegame` sample. In this case, we have provided a convenient script [build.ps1](./build.ps1) that will build and package both projects (`wrapper` and `fakegame`). Script will create a `drop` folder with a .zip file containing the required files. Make sure you define a unique name for your zip file example to avoid issues in PlayFab.
```

Comment:
this should be unique compared to the assets of a title right? Can you elaborate on the issues?